frontend/src/components/GroupView/groupview-sizes.scss

```diff
@@ -5,6 +5,11 @@ $progressbar-height: 80px;
 $sidebar-width: 80px;
 $sidebar-header-height: 24px;
 $sidebar-header-margin: 14px;
+$sidebar-personal-button-height: 15vh;
+$sidebar-group-manager-height: 85vh;
+$add-button-height: 80px;
+$expanding-div-base-height: 75px;
+$links-height: 62px;
```

Comment:
can we make these all divisible by 4. Random numbers don't look good

Reply:
What's the thinking behind making it divisible by 4? Just convention?

Reply:
some mature design system like material design has this systematic guideline: https://material.io/design/layout/spacing-methods.html#spacing

Having some random number like 62 is usually a sign that you tweaked the pixel value until it looks somehow correct, but without any systematic principle behind it. If we have those unprincipled number all over the place, it would be impossible fo enforce design consistency.